db/compaction_picker.cc

```diff
@@ -1373,6 +1356,39 @@ bool LevelCompactionBuilder::PickFileToCompact() {
   return start_level_inputs_.size() > 0;
 }
 
+bool LevelCompactionBuilder::PickL0ToBaseCompaction() {
```

Comment:
we could expand toward the left as long as compaction-bytes-per-deleted-L0-file decreases, like we do in intra-L0 compaction. That'd solve both problems. Do you prefer that?